src/components/Notification/Notification.scss

```diff
@@ -4,12 +4,12 @@
 
 .notificationCard {
   display: flex;
-  margin: 5px;
+  padding: 10px;
   word-break: break-all;
-  transition: transform 100ms;
-
+  transition: background-color 200ms;
```

Comment:
you should read https://www.creativebloq.com/advice/tips-for-css-animation and https://csstriggers.com/ what and how animate 

tldr your code is ok but could be better